.env

```diff
@@ -115,4 +115,6 @@ TRANSCRIBE_SERVER_URLS=
 # e.g. SERVER_URLS="http://1.2.3.4:8000,http://4.3.2.1:8000"
 DIARIZE_SERVER_URLS=
 #
+# Timeout for connection and waiting for result from remote diarize server. 300 sec is default value
+REMOTE_DIARIZE_SERVER_REQUEST_TIMEOUT_SEC=300
```

Comment:
Just `REMOTE_DIARIZATION_SERVER_REQUEST_TIMEOUT=300`

to keep things consistent